.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v**'
```

Comment:
```suggestion
on:
  release:
    types: [created]
```

This is triggered by tags, etc too.

Reply:
Are you sure that event makes sense?

I thought it's for actual releases - e.g. https://github.com/submariner-io/submariner-operator/releases

In such a case it will *only* trigger for tags but never for merge to the master branch..

Please correct me if I'm understanding this wrong

Reply:
I think you're right. I forgot of master